### DISCUSSION TITLE
how to get rid of keybinds message window on startup

### DISCUSSION THREAD

**Discussion by Asteroth2018** [YaLTeR/niri#629] [Ideas]
hello i know i might sound dumb but how to get rid of keybinds message window on startup. In the config file its not in spawn at startup section and i cant seem to find it anywhere

**Comment by YaLTeR** [comment]
Hey, https://github.com/YaLTeR/niri/wiki/Configuration:-Miscellaneous#hotkey-overlay

**Reply by cyuria** [reply]
Off memory I think I had a similar question and had to search a bit for the setting when I first started using niri. It might be a good idea to add it to the FAQ in the wiki or in the hotkey overlay itself.

**Reply by YaLTeR** [reply]
Good idea indeed. Added in da53e79d0718f0864f34da4da1de656eccfaf156
